### PR TITLE
Update site for v20.1.0 release

### PR DESCRIPTION
**Pull request by jseldess**

### CODE REVIEW

File: releases/v20.1.0.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+**Bulk Import Improvements** | The [`IMPORT`](../v20.1/import.html) and [`IMPORT INTO`](../v20.1/import-into.html) statements now support bulk importing [from Avro files](../v20.1/migrate-from-avro.html) and [from local storage](../v20.1/import.html#import-a-table-from-a-local-file). The former makes it easier to migrate from systems like Spanner that export data in the Avro format, and the latter makes it possible to import without cloud storage. The new [`cockroach nodelocal upload`](../v20.1/cockroach-nodelocal-upload.html) command can be used to upload a file to a node's external IO directory for local import.<br><br>Also, [paused](../v20.1/pause-job.html) imports, when [resumed](../v20.1/resume-job.html), now continue from their internally recorded progress instead of starting over.
```
Comment (mwang1026) [inline releases/v20.1.0.md:69]: nit. You could always import from nodelocal, it's just now easier to get those files to nodelocal from your "local" client (previously you had to SSH into the node and `scp` or something the file over)

What are your feelings on the usage of the word "local"? Local implies to me the machine you're typing on--if you SSH into a node I wouldn't personally consider that local. Do you have any thoughts on that @lnhsingh ? We seem to use local a bunch of places so I don't want to necessarily trigger a wholesale change across docs.

File: releases/v20.1.0.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+### Enterprise features
+
+These features require an [enterprise license](../v19.2/enterprise-licensing.html). Register for a 30-day trial license [here](https://www.cockroachlabs.com/get-cockroachdb/), or consider testing enterprise features locally using the [`cockroach demo`](../v19.2/cockroach-demo.html) CLI command, which starts an in-memory CockroachDB cluster with a temporary enterprise license pre-loaded. [CockroachCloud clusters](https://cockroachlabs.cloud/) also include all enterprise features.
+
+Feature | Description
+--------|------------
+**Full-cluster backups** | You can now use CockroachDB's `BACKUP` feature to [back up an entire cluster's data](../v20.1/backup.html#backup-targets), including all configuration and system information such as [user privileges](../v20.1/authorization.html#privileges), [zone configurations](../v20.1/configure-replication-zones.html), and [cluster settings](../v20.1/cluster-settings.html). In rare disaster recovery situations, you can use the `RESTORE` feature to [restore a full cluster backup to a new cluster](../v20.1/restore.html#full-cluster). Restoring a full cluster backup to an existing cluster is not supported.
```
Comment (mwang1026) [inline releases/v20.1.0.md:95]: IMO should be full-cluster backup and restore as the title--it's the restore part that's important to!

File: releases/v20.1.0.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+**Encrypted backups** | You can now use an encryption key to [encrypt data in Enterprise `BACKUP` files](../v20.1/backup.html#encrypted-backups), and to [decrypt the data upon `RESTORE`](../v20.1/restore.html#restore-from-an-encrypted-backup).
```
Comment (mwang1026) [inline releases/v20.1.0.md:96]: If we want to be really pedantic, the option is call `encryption_passphrase` so maybe "You can now use an encryption passphrase to..."?

File: releases/v20.1.0.md (hunk 1 above)
Comment (jseldess) [reply releases/v20.1.0.md:69]: Thanks, @mwang1026. I made some changes. PTAL. 

I agree that we overload the term "local". Initially, it was used to just contrast running cockroach on your computer vs on a server. We've probably outgrown that. We're likely to take a fresh look at our getting started docs very soon in any case.

File: releases/v20.1.0.md (hunk 2 above)
Comment (jseldess) [reply releases/v20.1.0.md:95]: Done.

File: releases/v20.1.0.md (hunk 3 above)
Comment (jseldess) [reply releases/v20.1.0.md:96]: Done.

**Comment by jseldess** [comment]
___
*[releases/v20.1.0.md, line 59 at r1](https://reviewable.io/reviews/cockroachdb/docs/7198#-M5etvnU1KFbz3QZxAzr:-M5hLXET3hs4DX98jBba:b-oaed3w) ([raw file](https://github.com/cockroachdb/docs/blob/fd3ae6547526f7ff81d5ef631e82e711b8b3819b/releases/v20.1.0.md#L59)):*
<details><summary><i>Previously, Amruta-Ranade (Amruta Ranade) wrote…</i></summary><blockquote>

- CockroachCloud pricing is now available on the [pricing page](https://www.cockroachlabs.com/pricing/).
- Free trials of CockroachCloud are now available. [Contact us](https://www.cockroachlabs.com/contact-sales/) to request a trial code.
- CockroachCloud now supports VPC peering for clusters running on GCP. [Contact us](https://support.cockroachlabs.com/hc/en-us) to set up a VPC peering-enabled CockroachCloud cluster.

cc @kannanlakshmi in case I missed something.
</blockquote></details>

Thanks, @amruta-ranade. I'll wait for confirmation from @kannanlakshmi and then write these up.

**Comment by Amruta-Ranade** [comment]
> > _[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/7198)_ status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @Amruta-Ranade, @jseldess, @lnhsingh, and @mwang1026)
> 
> _[releases/v20.1.0.md, line 59 at r1](https://reviewable.io/reviews/cockroachdb/docs/7198#-M5etvnU1KFbz3QZxAzr:-M5hXwtj7QjhiNRbckEw:b2uheq2) ([raw file](https://github.com/cockroachdb/docs/blob/fd3ae6547526f7ff81d5ef631e82e711b8b3819b/releases/v20.1.0.md#L59)):_
> 
> Previously, jseldess (Jesse Seldess) wrote…
> This LGTM. @Amruta-Ranade do we want to add the edit profile stuff here or do you think its not important at the top level? I am happy to defer to you here.

I don't think it qualifies as a top-level callout for the 20.1 release.

**Comment by mwang1026** [comment]
IO parts LGTM. Thanks!

**Comment by mwang1026** [comment]
I don't think we need to call out changefeed features, given that they weren't an area of focus for 20.1

File: releases/v20.1.0.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+**Nested Transactions** | CockroachDB now supports the nesting of transactions using savepoints. These [nested transactions](../v20.1/transactions.html#nested-transactions), also known as sub-transactions, can be rolled back without discarding the state of the entire surrounding transaction. This can be useful in applications that abstract database access using an application development framework or ORM. Different components of the application can operate on different sub-transactions without having to know about each others' internal operations, while trusting that the database will maintain isolation between sub-transactions and preserve data integrity.
+**Savepoint Naming** |  Savepoints for [client-side transaction retries](../v20.1/advanced-client-side-transaction-retries.html) default to using the name `cockroach_restart`, but the [savepoint name can now be customized](../v20.1/savepoint.html#customizing-the-retry-savepoint-name) to support applications that require different naming. For example, this may be necessary when using an ORM.
```
Comment (rmloveland) [inline releases/v20.1.0.md:74]: This section can be removed.  We added the ability to customize the savepoint name (for "retry savepoints" only) in 19.1:

https://www.cockroachlabs.com/docs/v19.1/advanced-client-side-transaction-retries.html#customizing-the-savepoint-name

**Comment by jseldess** [comment]
___
*[releases/v20.1.0.md, line 80 at r4](https://reviewable.io/reviews/cockroachdb/docs/7198#-M6BaF8vA0Kfs0EGNC4o:-M6Bq7N689T-9jm5jg-I:b-joszi4) ([raw file](https://github.com/cockroachdb/docs/blob/5c67b57e7b925c0a2fe1675ab45b31b16adf737b/releases/v20.1.0.md#L80)):*
<details><summary><i>Previously, awoods187 (Andy Woods) wrote…</i></summary><blockquote>

do we need to list this as experimental in the title?
</blockquote></details>

I don't think so. If they click through, they'll see that designation clearly.

**Comment by jseldess** [comment]
___
*[releases/v20.1.0.md, line 90 at r4](https://reviewable.io/reviews/cockroachdb/docs/7198#-M6BaVXA8-K-jBR-O-om:-M6BqH0o81LcRPfSRTpO:b-r41d49) ([raw file](https://github.com/cockroachdb/docs/blob/5c67b57e7b925c0a2fe1675ab45b31b16adf737b/releases/v20.1.0.md#L90)):*
<details><summary><i>Previously, awoods187 (Andy Woods) wrote…</i></summary><blockquote>

should we be calling out cc here?
</blockquote></details>

We are... See the last sentence. Are you saying we shouldn't?

**Comment by jseldess** [comment]
___
*[releases/v20.1.0.md, line 65 at r4](https://reviewable.io/reviews/cockroachdb/docs/7198#-M6B_rHgEMXx6pcHF1Gi:-M6C6cdS85BpD6AKcEdC:bk8s3rl) ([raw file](https://github.com/cockroachdb/docs/blob/5c67b57e7b925c0a2fe1675ab45b31b16adf737b/releases/v20.1.0.md#L65)):*
<details><summary><i>Previously, awoods187 (Andy Woods) wrote…</i></summary><blockquote>

i dont quite understand the order of the features. It's not alphabetical or by product area/code area. Is it roughly in importance order? i think an outside reader may be confused on this. I personally would like the things grouped in user journey areas (e.g. put schema migrations like schema changes using jobs and primary key changes together) or alphabetically.
</blockquote></details>

I agree. The ordering is intuitive, so there's really no ordering! I'll try adding areas.

**Comment by jseldess** [comment]
___
*[releases/v20.1.0.md, line 87 at r4](https://reviewable.io/reviews/cockroachdb/docs/7198#-M6BpUp47rxwd-LTOdos:-M6D31nzCn8znRot6yw1:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/5c67b57e7b925c0a2fe1675ab45b31b16adf737b/releases/v20.1.0.md#L87)):*
<details><summary><i>Previously, awoods187 (Andy Woods) wrote…</i></summary><blockquote>

I'm not sure if we should have a section on newly supported tools or if we should fold it into this section as a line item. We added support for django, jooq, ponyorm, and peewee. In future releases, we are likely to have GUIs and other tools beyond just ORMs as well.
</blockquote></details>

Done.

File: v20.1/upgrade-cockroach-version.md [hunk 5]
```diff
@@ -259,7 +267,7 @@ Once you are satisfied with the new version:
     > RESET CLUSTER SETTING cluster.preserve_downgrade_option;
     ~~~
 
-3. Update your application or tooling to re-enable schema changes that were disallowed during the upgrade process. **Note that this applies only to upgrades from 19.2 to 20.1. It does not apply to other upgrades.**
+3. Update your application or tooling to re-enable schema changes that were disallowed during the upgrade process. Note that this applies only to upgrades from 19.2 to 20.1. It does not apply to other upgrades.
```
Comment (lnhsingh) [inline v20.1/upgrade-cockroach-version.md:270]: Change `19.2 to 20.1` to `v19.2 to v20.1`

File: v20.1/upgrade-cockroach-version.md [hunk 6]
```diff
@@ -38,25 +38,31 @@ Verify the overall health of your cluster using the [Admin UI](admin-ui-access-a
...
+Once all nodes are running v20.1, but before the upgrade has been finalized, any schema changes still running will stop making progress, but [`SHOW JOBS`](show-jobs.html) and the [**Jobs** page](admin-ui-jobs-page.html) in the Admin UI will show them as running until the upgrade has been finalized. During this time, it won't be possible to manipulate these schema changes via [`PAUSE JOB`](pause-job.html)/[`RESUME JOB`](resume-job.html)/[`CANCEL JOB`](cancel-job.html) statements. Once the upgrade has been finalized, these schema changes will run to completion.
```
[the remark this answers is not in the document]
Comment (jseldess) [reply v20.1/upgrade-cockroach-version.md:52]: I agree this sentence is on the long side, but the but feels important here.

File: v20.1/upgrade-cockroach-version.md [hunk 7]
```diff
@@ -38,25 +38,31 @@ Verify the overall health of your cluster using the [Admin UI](admin-ui-access-a
...
+
+Note that this behavior is specific to upgrades from 19.2 to 20.1; it does not apply to other upgrades.
```
[the remark this answers is not in the document]
Comment (jseldess) [reply v20.1/upgrade-cockroach-version.md:54]: Done.

File: v20.1/upgrade-cockroach-version.md (hunk 5 above)
Comment (jseldess) [reply v20.1/upgrade-cockroach-version.md:270]: Done.

**Pull request merged**